reactRouter/clientList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import List from './list.jsx';
+import request from 'sync-request';
+import AddList from './addList.jsx';
+import { Router, Route, Link, IndexRoute, hashHistory, browserHistory, DefaultRoute } from 'react-router'
+class ClientList extends React.Component { 
+  constructor(props) {
+    super(props);
+    this.state = {clientList: []}
+  }
+  
+
+  componentWillMount() {
+    var clientList;
+    var res = request('GET', 'http://localhost:8081/clients');
```

Comment:
@VARTIKASINGH1234 Always keep the API URL i.e 'http://localhost:8081/clients' in the separate file. So In future If URL would change then, we don't need to modify the URL name in each file.